tests/php/test_class.jetpack-sync.php

```diff
@@ -0,0 +1,276 @@
+<?php
+// phpunit --filter test_sync_
+class WP_Test_Jetpack_Sync extends WP_UnitTestCase {
+
+	protected $_globals;
+
+	public function setUp() {
+		require_once dirname( __FILE__ ) . '/../../class.jetpack-rest-sync.php';
+		parent::setUp();
+
+		Jetpack_Rest_Sync::init();
+	}
+
+	public function tearDown() {
+		parent::tearDown();
+	}
+
+	public function test_sync_new_post() {
+		$new_post = array(
```

Comment:
I think we could extract this to a helper as we are repeating it a lot of times.